cluster/client/authorize.go

```diff
@@ -0,0 +1,96 @@
+// Code generated by goagen v1.3.0, DO NOT EDIT.
```

Comment:
if the repo is named `fabric8-cluster-client`, then can we avoid the `cluster/client` pkg here, i.e, just have all go files at the root of the repo ? or if this causes troubles, in a package named `pkg` which is where public code usually goes (as opposed to `internal`) ?

Reply:
The `pkg` pattern just indicates that all subdirs are public. It's not intended to be used as a package. So, while it's OK to have `/pkg/auth` or `/pkg/cluster` it really doesn't look good when you use `/pkg`:
```
import (
    "github.com/fabric8-services/fabric8-cluster-client/pkg"
)
...
    c := pkg.NewClient()
```

`pkg` dir pattern is not about organizing source code in some "root" package. It's about distinguishing public vs internal packages (and btw we don't use this pattern in OSIO at all which is fine).

I would suggest moving all the code to some meaningful package name. What about `"github.com/fabric8-services/fabric8-cluster-client/cluster"`?
```
import (
    "github.com/fabric8-services/fabric8-cluster-client/cluster"
    "github.com/fabric8-services/fabric8-auth-client/auth"
    "github.com/fabric8-services/fabric8-wit-client/wit"
)
...
    clusterCln := cluster.NewClient()
    authCln := auth.NewClient()
    witCln := wit.NewClient()
```

Reply:
ok, fair point @alexeykazakov. The code you proposed above looks clean 👍

Reply:
@dipak-pawar can you apply the change as @alexeykazakov suggested, please?

Reply:
@xcoulon: yes, I can.